task9/task9/Program.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task9
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите необходимое количество чисел в массиве");
+
+            int n = int.Parse(Console.ReadLine());
+
+            if (n < 0)
+            {
+                PrintWithDivision("Отрицательная длина массива! Какая подлость!");
+                Console.ReadKey();
+                return;
+            }
+
+            PrintWithDivision("Введите два целых числа в диапазоне от -20 до 20");
+            Console.WriteLine("(первое число должно быть меньше второго)");
+            
+            int a = int.Parse(Console.ReadLine());
+            int b = int.Parse(Console.ReadLine());
+
+            if ((Math.Abs(a) > 20) || (Math.Abs(b) > 20))
+            {
+                PrintWithDivision("Как минимум одно из чисел находится вне допустимого диапазона");
+                Console.ReadKey();
+                return;
+            }
+
+            if (a >= b)
+            {
+                PrintWithDivision("Что непонятного в фразе \"первое число должно быть меньше второго\"?");
+                Console.ReadKey();
+                return;
+            }
+
+            int[] array = new int[n];
+            Random rand = new Random();
+
+            for (int i = 0; i < array.Length; i++)
+                array[i] = rand.Next(a, b);
```

Comment:
Чтобы _b_ тоже могло появиться в массиве (неравенство нестрогое!) нужно так
```suggestion
                array[i] = rand.Next(a, b + 1);
```